packages/desktop-client/src/components/gocardless/GoCardlessLink.tsx

```diff
@@ -1,23 +1,20 @@
 import React from 'react';
 
-import { Modal } from '../common/Modal';
+import { Modal, ModalHeader } from '../common/Modal';
 import { Paragraph } from '../common/Paragraph';
 import { View } from '../common/View';
 
 export function GoCardlessLink() {
-  window.close();
```

Comment:
I think we want to keep this line. Otherwise the window won't automatically close after linking.

The flow is:
1. initialise the linking
2. that opens the linking in popover
3. the user follows bank login there
4. finally the popover redirects back to actual
5. we close the popover (this line)
6. the root window is visible again and linking is success

Reply:
Nice catch! Thanks for pointing that out. Updated :)